### PR TITLE
Update office-365-groups-expiration-policy.md

### PR DESCRIPTION
**Pull request by grtaylo**
Added additional detail regarding activity-based automatic renewal.

### CODE REVIEW

**Comment by MikePlumleyMSFT** [comment]
@grtaylo I merged part of this, but many of the behind-the-scenes details are things we don't normally cover in articles like this. If there's important info that you feel is missing from this article, please contact me internally.

File: microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ You can set the policy for all of your groups, only selected groups, or turn it
 
 ![Screenshot of Groups expiration settings in Azure Active Directory](../../media/azure-groups-expiration-settings.png)
 
+> [!IMPORTANT]
+> Activity-based utomatic group renewal is now enabled by default, and cannot be administratively disabled. For more information on automatic renewal see [Activity-based automatic renewal(https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/groups-lifecycle)
```
Comment (abgupta) [inline microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md]: @grtaylo , typo in "utomatic".

File: microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md [hunk 2]
```diff
@@ -73,7 +76,20 @@ Group owners will only be notified via email. If the group was created via Plann
...
+If for some reason none of the owners or admins renew the group before it expires, and automatic renewal does not occur due to the group not meeting the requirements to be automatically renewed, the admin can still restore the group for up to 30 days after expiration. For details see: [Restore a deleted Microsoft 365 group](https://docs.microsoft.com/microsoft-365/admin/create-groups/restore-deleted-group).
+
+## How automatic renewal fits into the group expiration timeline
+
+Automatic renewal checks occur starting 40 days in advance from the group's expiration date. The service will continually check for activity from this point until the group expires, is manually renewed, or meets the requirements for automatic renewal and is thus renewed. If the service determines the group has been active at any point during the group's lifecycle, the service will begin the automatic renewal process. For example, if activity occurred 60 days from expiration, the activity check performed 40 days from expiration would indicate the group is active, and the automatic group renewal process would immediately begin.
```
Comment (abgupta) [inline microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md]: Can we rephrase this as follows:
Automatic renewal checks occur continuously during the Group's lifecycle. If there is any activity in the group prior to 30 days of expiry then the group will be auto-renewed and group owners will not receive any e-mail notifications for manual group renewal.

File: microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md [hunk 3]
```diff
@@ -73,7 +76,20 @@ Group owners will only be notified via email. If the group was created via Plann
...
+
+> [!NOTE]
+> The service that checks to see if a group should be automatically renewed can take up to 48 hours to process and finish the automatic renewal procedure. In very rare cases, this can lead to the group failing to automatically renew, if no activity meeting the automatic renewal requirements occurs within the group's lifecycle until < 48 hours of the group's expiration time.
+
+## How email notifications work with automatic group renewal
+
+If a group is renewed prior to the scheduled time for an expiration reminder email being sent to the group owners or administrators, then further reminder emails will not be sent. For example, if the group is automatically renewed 40 days from expiration, then the 30 day, 15 day, and one day reminder emails will not be sent. If the group was renewed 12 days from expiration, both the 30 day and the 15 day reminder emails would be sent, but not the one day reminder.
```
Comment (abgupta) [inline microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md]: Change 40 days to 30 days

File: microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md [hunk 4]
```diff
@@ -73,7 +76,20 @@ Group owners will only be notified via email. If the group was created via Plann
...
+
+## How automatic renewal fits into the group expiration timeline
+
+Automatic renewal checks occur starting 40 days in advance from the group's expiration date. The service will continually check for activity from this point until the group expires, is manually renewed, or meets the requirements for automatic renewal and is thus renewed. If the service determines the group has been active at any point during the group's lifecycle, the service will begin the automatic renewal process. For example, if activity occurred 60 days from expiration, the activity check performed 40 days from expiration would indicate the group is active, and the automatic group renewal process would immediately begin.
+
+This also means that if no activity is detected within the group's lifecycle when, 40 days from expiration, the automatic renewal service first begins searching for activity meeting the requirements for automatic renewal, the group may not be automatically immediately renewed 40 days from expiration. For example, if the group is inactive until 35 days from renewal, the automatic renewal process would not occur 40 days from expiration, and would instead begin 35 days from renewal.
```
Comment (abgupta) [inline microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md]: Can we rephrase as follows:
On the other hand if there is no activity in the group 30 days prior to expiry then group owners will receive an e-mail requesting manual renewal of the group. Similar e-mail notifications will also be sent 15 days and 1 day prior to expiry. Simultaneously, auto-renewal checks will continue to occur and will auto-renew the group anytime an activity is detected up to 48 hours prior to group expiry.
